anarchy-runner/Dockerfile

```diff
@@ -31,6 +31,7 @@ RUN yum install -y \
  && ln -s pip${PYTHON_VERSION} pip3 \
  && ln -s pydoc${PYTHON_VERSION} pydoc3 \
  && pip3 install -r /anarchy-runner/requirements.txt --upgrade \
+ && sed -i -e 's|^\( *\)# If there are multiple matches, prefer non-List kinds|\1# Hotfix openshift template discovery\n\1results = [ result for result in results if result.group_version != "template.openshift.io/v1" or result.name != "processedtemplates" ]|' /usr/local/lib/python3.7/site-packages/openshift/dynamic/{client,discovery}.py \
```

Comment:
That long line is very ugly, even if hot fix.

What about creating a .patch file, add it with `ADD` and use the `patch` command instead ?

Reply:
I'll look into what this will take to use a patch inside of a docker file.